desk/app/tenna.hoon

```diff
@@ -164,7 +166,9 @@
   ?+    path
     (on-watch:def path)
       [%frontend ~]
-    `this
+    :_  this
+    :~  (active:vita-client bowl)
```

Comment:
we have to give this card in a place that indicates user activity.  this is a self-poke. it only gets sent out if the wrapper is enabled and enough time has passed since the last poke.